src/commands/auth/web/login.ts

```diff
@@ -26,6 +26,12 @@ export default class Login extends SfdxCommand {
   public static aliases = ['force:auth:web:login'];
 
   public static readonly flagsConfig: FlagsConfig = {
+    browser: flags.string({
+      char: 'b',
+      description: messages.getMessage('browser'),
+      options: ['chrome', 'edge', 'firefox'], // These are ones supported by "open" package
```

Comment:
since `open` only supports these three browsers, this flag can be an enum like this one:
https://github.com/salesforcecli/plugin-data/blob/2c37ba80f2093b602bfc4e18980f4516525056a2/src/commands/force/data/soql/query.ts#L267

The benefit of `flags.string` over `flags.enum` is that when you run `sfdx auth:web:login --help` it will show the user that only chrome, edge or firefox are valid values.

Reply:
Thank you fro the review, your suggestions have been implemented.